unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/TaskbarHUD/IntercomButton.cs

```diff
@@ -0,0 +1,15 @@
+using DCL.Interface;
+using UnityEngine;
+using UnityEngine.UI;
+public class IntercomButton : MonoBehaviour
```

Comment:
Shouldnt this follow MVP pattern?

Reply:
Refactored the behaviour in a proper way